### PR TITLE
SC 1498 added the env variables for cert-service to enable license path

### PR DESCRIPTION
**Pull request by Hari-stackroute**

### CODE REVIEW

File: ansible/roles/stack-sunbird/templates/sunbird_cert-service.env [hunk 1]
```diff
@@ -5,4 +5,6 @@ AZURE_STORAGE_KEY={{sunbird_azure_storage_account}}
 sunbird_cert_domain_url={{proto}}://{{proxy_server_name}}
 sunbird_cert_enc_service_url=http://enc-service:8013
 download_link_expiry_timeout=600
-es_conn_info={{sunbird_es_host}}:{{sunbird_es_http_port}}
\ No newline at end of file
+es_conn_info={{sunbird_es_host}}:{{sunbird_es_http_port}}
```
Comment (SMYALTAMASH) [inline ansible/roles/stack-sunbird/templates/sunbird_cert-service.env]: can you check the correct variable value for this @Hari-stackroute

Comment (SMYALTAMASH) [reply ansible/roles/stack-sunbird/templates/sunbird_cert-service.env]: @maheshkumargangula can you please suggest the correct value for this?

Comment (SMYALTAMASH) [reply ansible/roles/stack-sunbird/templates/sunbird_cert-service.env]: @Hari-stackroute change the value to "{{groups['es']|join(':9200,')}}:9200"

Comment (Hari-stackroute) [reply ansible/roles/stack-sunbird/templates/sunbird_cert-service.env]: @SMYALTAMASH It is done, please check and merge.

**Pull request merged**